config/routes.rb

```diff
@@ -15,7 +15,7 @@
   end
 
   resources :messages, only: [:create] do
-    patch :read
+    resource :read, only: :create
```

Comment:
You can organize the files better with one set of changes. Update the route to be:

```
resources :messages, only: [:create] do
  scope module: :messages do
    resource :read, only: :create
  end
end
```

and then rename the file to be `app/controllers/messages/reads_controller.rb` and the class inside that file to be `class Messages::ReadsController`.